IdentityCore/src/MSIDError.m

```diff
@@ -99,10 +99,30 @@ MSIDErrorCode MSIDErrorCodeForOAuthError(NSString *oauthError, MSIDErrorCode def
     {
         return MSIDErrorServerAccessDenied;
     }
-    
+    if (oauthError && [oauthError caseInsensitiveCompare:@"tokenTransferFailedOTC"] == NSOrderedSame)
```

Comment:
Why is this error code inconsistent with other codes? (others have underscores)

Reply:
This was implemented by the MMX folks as part of the codev model while working in our UX codebase and wasn't reviewed by someone in MSA's team. They have a plan to change this to a standard oauth-like errors (e.g.  something like "transfer_token_failed"). Since MMX team already has a dependency on the older error code, we would need to support both for some time, before the MMX team also moves to new error code 
-- 
This was the reply from Guruprasad Balkrishna Aphale